Neos.ContentRepository.Core/Classes/Projection/ContentStream/ContentStreamProjection.php

```diff
@@ -184,7 +182,7 @@ public function apply(EventInterface $event, EventEnvelope $eventEnvelope): void
             ContentStreamWasReopened::class => $this->whenContentStreamWasReopened($event, $eventEnvelope),
             ContentStreamWasRemoved::class => $this->whenContentStreamWasRemoved($event, $eventEnvelope),
             DimensionShineThroughWasAdded::class => $this->whenDimensionShineThroughWasAdded($event, $eventEnvelope),
-            default => throw new \InvalidArgumentException(sprintf('Unsupported event %s', get_debug_type($event))),
+            default => $event instanceof EmbedsContentStreamId || throw new \InvalidArgumentException(sprintf('Unsupported event %s', get_debug_type($event))),
```

Comment:
If we'd check for both interfaces in line 167, this change could be reverted I guess!?